js-mentoring-program-master/tasks/json/parser.js

```diff
@@ -13,4 +13,20 @@
  * where 077b1523-56f0-492a-b954-1269e3acc191 - is the value of the name attribute from the ./test.json file
  * Please NOTE, that you should omit the .html extension
  * Please do the task in the scope of the file. Create a pull request a share it with your mentor.
- */
\ No newline at end of file
+ */
+const fs = require('fs');
+const testJson = require('./test.json');
+ 
+const entries = testJson.list.entries;
+ 
+const parsed = entries.map((entry) => {
+    const docId = `http://doc.epam.com/${entry.entry.name.slice(0, -5)}`;
```

Comment:
You don't need to declare and return variable, just return this expression instead.